packages/core/src/browser/core-preferences.ts

```diff
@@ -53,10 +55,12 @@ export const corePreferenceSchema: PreferenceSchema = {
         },
         'workbench.colorTheme': {
             type: 'string',
+            default: applicationConfig?.defaultTheme || 'dark',
```

Comment:
It would be better to make `defaultTheme` and `defaultIconTheme` non-optional, and use defaults the same way as we do for the `applicationName`. I do not want to have hardcoded `'dark'` and `'none'` all over the application. Everything should be part of the `DEFAULT` application props.

https://github.com/eclipse-theia/theia/blob/af98a8eb169e7dceb04d51d53249a0c6eec6b328/dev-packages/application-package/src/application-props.ts#L75-L91

Please change.

Reply:
good idea